include/range/v3/utility/basic_iterator.hpp

```diff
@@ -352,18 +352,22 @@ namespace ranges
             using box<T>::get;
         };
 
-        /// \cond
-        namespace _basic_iterator_
-        {
-        /// \endcond
+#if RANGES_BROKEN_CPO_LOOKUP
+        // Workaround https://bugs.llvm.org/show_bug.cgi?id=37556
```

Comment:
All of these `// Workaround BUG#` comments should be centralized into the definition of `RANGES_BROKEN_CPO_LOOKUP` in `config.hpp` instead of being spammed throughout the source.